web-tools.md

```diff
@@ -117,4 +117,5 @@
 [Marcdown](https://liyasthomas.github.io/marcdown) | A lightweight markdown viewer and editor.
 [HackerRank](https://www.hackerrank.com/) | An awesome platform to help you enhance your skills by providing problems to solve in a wide range of Computer Science domains.
 [Vimium](https://github.com/philc/vimium) | The Hacker's Browser. Forget about your mouse. Use VIM to navigate on your browser.
-[Sejda PDF Editor](https://www.sejda.com/pdf-editor) | The best online PDF editor tool, easily edit text, put whiteout over text which is not needed and much more.
\ No newline at end of file
+[Sejda PDF Editor](https://www.sejda.com/pdf-editor) | The best online PDF editor tool, easily edit text, put whiteout over text which is not needed and much more.
+[Tinkercad](https://www.tinkercad.com/) | Tinkercad is a free, easy-to-use app for 3D design, electronics, and coding. It's used by teachers, kids, hobbyists, and designers to imagine, design, and make anything!
```

Comment:
is it a web tool or desktop tool?

Reply:
Yes, it is a web tool. I always used it on navigators, and I never heard of a desktop app version for this.